config/locales/en/general.yml

```diff
@@ -573,8 +573,7 @@ en:
         poster_title: "Do not keep looking,"
         poster_subtitle: "back me up! ;)"
         intro_text:
-          "<strong>I am participating in %{org}</strong> with my own citizen proposal and only if you also add you can I achieve the
-          necessary support to make the city we all want."
+          "<strong>I am participating in %{org}</strong> with my own citizen proposal and only if you add to it i will achieve the necessary support to make the city we all want."
```

Comment:
"only if you add to it" sounds a bit strange to me :thinking:. How about:

```
"<strong>I am participating in %{org}</strong> with my own citizen proposal and only with your support can we make the city we all want"
```

Not sure about using the word "city", since CONSUL can be used in other contexts :thinking:, but I guess we can leave it the way it was :relieved:.

Reply:
Much better @javierm! Thanks ☺️